taf/tools/targets/__init__.py

```diff
@@ -42,7 +42,7 @@ def add_repo_command():
         `namespace1\\repo1`, the target's path will be set to `E:\\examples\\root\\namespace1\\repo1`.""")
     @catch_cli_exception(handle=TAFError)
     @click.option("--path", default=".", help="Authentication repository's location. If not specified, set to the current directory")
-    @click.option("--target-name", default=None, help="Namespace prefixed name of the target repository")
+    @click.option("--target-name", required=True, default=None, help="Namespace prefixed name of the target repository")
```

Comment:
We typically use an argument instead of an option that needs to be provided. See other commands. I'd like to maintain consistency, so I'd rather use it here as well.

Reply:
Understood, sorry for the confusion. I was looking use the same declaration. I have now updated it properly.